tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -125,6 +126,22 @@ function renderGlobalModal(options?: Options) {
   return {...result, waitForModalToHide};
 }
 
+function renderGlobalDrawer(options?: Options) {
+  const result = render(<GlobalDrawer />, options);
+
+  /**
+   * Helper that waits for the drawer to be hidden from the DOM. You may need to
+   * wait for the drawer to be removed to avoid any act warnings.
+   */
+  function waitForDrawerToHide() {
+    return rtl.waitFor(() => {
+      expect(rtl.screen.queryByRole('complimentary')).not.toBeInTheDocument();
```

Comment:
Let's give it a label to so we can use the `name` option. Since I can imagine there are other complimentary elements.

Reply:
missed this the first time, thanks for following up